.doc_gen/templates/zonbook/sdk_chapter_template.xml

```diff
@@ -8,103 +8,151 @@
-        <titleabbrev id="{{.LanguageSlug}}_code_examples.titleabbrev">Code examples</titleabbrev>
+        <title id="{{.LanguageSlug}}_code_examples.title">{{$chapter_title}}</title>
+        <titleabbrev id="{{.LanguageSlug}}_code_examples.titleabbrev">{{$chapter_title_abbrev}}</titleabbrev>
         <abstract>
             <para>Code examples that show how to use {{.SdkEntity.Long}} with &AWS;.</para>
         </abstract>
     </info>
     <para>The following code examples show how to use {{.SdkEntity.Long}} with &AWS;.</para>
-    <para role="contents-abbrev">Code examples</para>
-    {{- if .CrossServiceExamples}}
-    <section id="{{.LanguageSlug}}_code_examples_cross_service" role="topic">
-        <info>
-            <title id="{{.LanguageSlug}}_code_examples_cross_service.title">Cross-service examples using {{.SdkEntity.Short}}</title>
-            <titleabbrev id="{{.LanguageSlug}}_code_examples_cross_service.titleabbrev">Cross-service examples</titleabbrev>
-            <abstract>
-                <para>The following code examples show how to use {{.SdkEntity.Short}} to combine
-                    &AWS; services.</para>
-            </abstract>
-        </info>
-        <para>The following code examples show how to use {{.SdkEntity.Long}} to combine
-            multiple &AWS; services.</para>
-        <para role="topiclist-abbrev">Examples</para>
-        {{- end}}
-        {{- range .CrossServiceExamples}}
-        <section id="{{.ExampleId}}_{{$.LanguageSlug}}_topic" role="topic">
-            <info>
-                <title id="{{.ExampleId}}_{{$.LanguageSlug}}_topic.title">{{.Title}}</title>
-                <titleabbrev id="{{.ExampleId}}_{{$.LanguageSlug}}_topic.titleabbrev">{{.TitleAbbrev}}</titleabbrev>
-            </info>
-            <xi:include href="file://AWSShared/code-samples/docs/{{.ExampleId}}_{{$.Language}}.xml"></xi:include>
-            {{- if .GuideTopic.Url}}
-            <para>For more information, see <ulink {{.GuideTopic.DocType}} url="{{.GuideTopic.Url}}">{{.GuideTopic.Text}}</ulink>.</para>
-            {{- end}}
-        </section>
-        {{- end}}
-        {{- if .CrossServiceExamples}}
-    </section>
-    {{- end}}
+    <variablelist role="termdef">
+        <para>The examples are divided into the following categories:</para>
+        <varlistentry>
+            <term>Single-service actions</term>
+            <listitem><para>Excerpts that illustrate how to write code to call individual service functions.</para></listitem>
+        </varlistentry>
+        <varlistentry>
+            <term>Single-service scenarios</term>
+            <listitem><para>Examples that show how to write code to accomplish a specific task with a single service.</para></listitem>
+        </varlistentry>
+        <varlistentry>
+            <term>Cross-service</term>
+            <listitem><para>Sample applications that combine multiple &AWS; services.</para></listitem>
```

Comment:
<listitem><para>Sample applications that work across multiple &AWS; services.</para></listitem>

Reply:
Done